csharp-basics/exercises/Collections/FlightPlanner/Program.cs

```diff
@@ -1,23 +1,138 @@
-        private const string Path = "../FlightPlanner/flights.txt";
+        private static readonly string Path = "..//..//flights.txt";
 
         private static void Main(string[] args)
         {
-            var readText = File.ReadAllLines(Path);
-            foreach (var s in readText)
+            var citysList = new List<string>(File.ReadAllLines(Path));
+            var citysFrom = new HashSet<string>();
+            var travelList = new HashSet<string>();
+
+            foreach (var citys in citysList)
+            {
+                var city = citys.Split('-');
+                citysFrom.Add(city[0]);
+            }
+
+            Console.WriteLine("What would you like to do:");
+            Console.WriteLine("To display list of the cities press 1");
+            Console.WriteLine("To exit program press #");
+            var choice = Console.ReadKey().KeyChar;
+            Console.WriteLine();
+
+            switch (choice)
             {
-                Console.WriteLine(s);
+                case '1':
+
+                    foreach (var citys in citysFrom)
+                    {
+                        Console.WriteLine(citys);
+                    }
+
+                    Console.WriteLine("select a city from which you would like to start: ");
+                    var input = Console.ReadLine();
+                    var startCity = String.Empty;
+
+                    foreach (var citys in citysList)
+                    {
+                        var city = citys.Split('-');
+                        city[0] = city[0].Replace(">", "");
+
+                        if (city[0].Contains(input))
+                        {
+                            startCity = city[0].Trim();
+                            travelList.Add(city[0]);
+                            Console.WriteLine(citys);
+                        }
+                    }
+
+                    string currentCity = "";
+
+                    while (startCity != currentCity)
+                    {
+                        Console.WriteLine(currentCity == ("") ? $"You are in {startCity}" : $"You are in {currentCity}");
+                        Console.WriteLine("What would you like to do:");
+                        Console.WriteLine("To display list of the cities press 1");
+                        Console.WriteLine("select a city press 2");
+                        Console.WriteLine("To exit program press 3");
+                        choice = Console.ReadKey().KeyChar;
+                        Console.WriteLine();
+
+                        if (choice == '1')
+                        {
+                            Console.WriteLine("Available flights for current city");
+
+                            if (currentCity == "")
+                            {
+                                string temp = input;
+
+                                foreach (var citys in citysList)
+                                {
+                                    var city = citys.Split('-');
```

Comment:
Šo tu dari arī iepriekš. Vagad izveidot atsevišķu mainīgo, kur ir jau vajadzīgie dati, lai nav jāatkāŗto darbības